index.css

```diff
@@ -2,16 +2,28 @@ body {
   font-family: "DM Sans", "Arial", sans-serif;
   margin: max(10vw, 20px);
 }
+
+.content_image{
+  width: 50%;
+}
+
 @media(max-width: 1100px){ 
   .container{ 
     padding-top: 1em;
   } 
+
+  .content_image{
+    width: 70%;
+  }
 } 
 
 @media(max-width: 515px){ 
   .container{ 
     padding-top: 6em;
   } 
+  .content_image{
```

Comment:
I don't think this part is necessary, because it adheres to the changes made to the larger sizes, I could be wrong.
But try taking this part out and see if it works.

Reply:
> I don't think this part is necessary, because it adheres to the changes made to the larger sizes, I could be wrong.
> But try taking this part out and see if it works.

The media queries are there to prevent it, but maybe i did something wrong. Anyway i'll revise it and try again. Thanks for replying!